gytheio-commons/src/test/java/org/gytheio/content/handler/DelegatingContentHandlerImplTest.java

```diff
@@ -179,10 +179,18 @@ public void testPutInputStream() throws Exception
     @Test
     public void testDelete() throws Exception
     {
-        delegatingHandler.delete(contentReferenceFile1b);
-        
-        File targetFile = new File(new URI(contentReferenceFile1b.getUri()));
-        assertFalse(targetFile.exists());
+        File sourceFile = new File(this.getClass().getResource(
+                    "/content-handler-b/file-b-1.txt").toURI());
+
+        ContentReference targetContentReferenceToDelete = getTextReference("/content-handler-b");
+        targetContentReferenceToDelete.setUri(targetContentReferenceToDelete.getUri() + "/file-b-3.txt");
+
+        delegatingHandler.putFile(sourceFile, targetContentReferenceToDelete);
```

Comment:
Usually we'd want this test method to only exercise the method under test, `delegatingHandler.delete` in this case, for more definitive test results rather than using other methods in the object under test to set things up as that could lead to confusing test results.

For example, if we had a bug in the `delegatingHandler.putFile` this test would fail as well as `testPutFile`.

It doesn't need to hold up the merge though.